### PR TITLE
completed all the html issues except clock

### PR DESCRIPTION
**Pull request by Pezzyace**
## Pull Request Type

What kind of change does this PR introduce?

[ ] Solved an issue/task

I completed the html easy, medium, and hard tasks (except the clock task)

### CODE REVIEW

File: tasks/css/medium/card/styles.css [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+
+.card{
+  width: 300px;
+  height: 300px;
+  background-color: #f8f9fa;
+  padding: 20px;
+  border: 1px solid #dee2e6;
+  border-radius: 5px;
+  box-shadow: 0px 2px 5px rgba(0,0,0,0.15);
+}
```
Comment (nikohoffren) [inline tasks/css/medium/card/styles.css:13]: Card looks good!

File: tasks/css/hard/card.css [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+}
+
+.card-front {
+  background-color: #ff9f00;
+}
+
+.card-back {
+  background-color: #4caf50;
+  transform: rotateY(180deg);
+}
```
Comment (nikohoffren) [inline tasks/css/hard/card.css:39]: Flipping animation seems to be working fine!

File: tasks/css/easy/styles.css [hunk 3]
```diff
@@ -1,3 +1,6 @@
 /* CSS - Easy */
 
 /* TODO: Implement red-class */
+.red{
+  color: red;
+}
```
Comment (nikohoffren) [inline tasks/css/easy/styles.css:6]: Looks good!

File: tasks/css/medium/fire/styles.css [hunk 4]
```diff
@@ -130,10 +150,30 @@ body {
...
+  height: 15px;
+  background-color: #ef5a00;
+  border-radius: 50%;
+  filter: drop-shadow(0 0 10px #d43322);
+  animation: particleUp 4s ease-out;
+  animation-iteration-count: infinite;
+  animation-fill-mode: both;
 }
 
 .fire-bottom .main-fire {
```
Comment (nikohoffren) [inline tasks/css/medium/fire/styles.css:179]: Fire animation looks good!

File: tasks/html/easy/index.html [hunk 5]
```diff
@@ -9,6 +9,7 @@
     </head>
     <body>
         <!-- TODO: Your task is to insert an h1 tag with the name "John Doe" -->
+        <h1>John Doe</h1>
         <h2>Contact Information:</h2>
         <a href="mailto:john.doe@example.com">Email me!</a>
     </body>
```
Comment (nikohoffren) [inline tasks/html/easy/index.html:15]: Looks good!

File: tasks/html/hard/index.html [hunk 6]
```diff
@@ -11,22 +11,63 @@
...
             <!-- TODO: Add footer -->
+            <div id="footer-links">
+                <a href="">About</a>
+                <a href="">Contact</a>
+                <a href="">Services</a>
+            </div>
+            <p>@ 2023 Fork, Commit, Merge. All rights reserved</p>
         </footer>
     </body>
 </html>
```
Comment (nikohoffren) [inline tasks/html/hard/index.html:73]: Code for the page looks good!

File: tasks/html/medium/index.html [hunk 7]
```diff
@@ -8,6 +8,21 @@
...
+                </label>
+                <div class="message">
+                    <p>Message</p>
+                    <textarea name="message" id="" cols="30" rows="10"></textarea>
+                </div>
+                <input type="submit" value="Submit">
+            </form>
         </div>
     </body>
 </html>
```
Comment (nikohoffren) [inline tasks/html/medium/index.html:28]: Form looks good!

**Review by nikohoffren — APPROVED** [review]
Files look good!

In the future, as it says on the README, you should make a new branch and new pull request for each of the tasks. That way it would be easier for me to review because there would not be any unrelated files and also you would get more PR:s merged. Now this counts only as one merged PR.

**Comment by nikohoffren** [comment]
## Merged

> This is an automated message from Fork, Commit, Merge [BOT].

Thank you for your contribution! Your pull request has been merged. The files have been reset for the next contributor.

### What's next?

If you're looking for more ways to contribute, I invite you to check out my other projects. Just click [here](https://github.com/nikohoffren?tab=repositories) to find more. These projects contain real issues that you can help resolve. You can also check out the [Influences](https://github.com/nikohoffren/fork-commit-merge#influences) section in the README to find more projects similar to this one.

Also please leave a star to this project if you feel it helped you, i would really appreciate it.

I look forward to seeing your contributions!

**Comment by Pezzyace** [comment]
Thank you.. Did my best

**Pull request merged**